commands/behold.js

```diff
@@ -0,0 +1,25 @@
+const Augur = require("augurbot"),
+    u = require("../utils/utils");
+
+const Module = new Augur.Module()
+    .addCommand({
+        name: "Behold",
+        description: "creates a doofenshritz behold meme, with the arguments becoming your bottom text. Small chance of fire.",
+        permissions: (msg) => msg.member && msg.channel.permissionsFor(msg.member).has(["ATTACH_FILES", "EMBED_LINKS"]),
+        process: (msg) => {
+            function randomNumber(min, max) {
+                return Math.round(Math.random() * (max - min) + min);
+            }
+            //Roll a D20. on a 20 get a squirell. On a 1, elmo burns. anything else you get heinz
+            function backgroundImage(override = randomNumber(1, 20)) {
+                switch (override) {
```

Comment:
While this works, having a function in the variable default gets kind of clunky. Recommend pushing that down to the `switch` like so:
```suggestion
            function backgroundImage(override) {
                switch (override || randomNumber(1, 20)) {
```
Not a big deal either way.